src/Avalonia.Base/Media/TileBrush.cs

```diff
@@ -74,6 +74,12 @@ public abstract class TileBrush : Brush, ITileBrush
         public static readonly StyledProperty<TileMode> TileModeProperty =
             AvaloniaProperty.Register<TileBrush, TileMode>(nameof(TileMode));
 
+        /// <summary>
+        /// Defines the <see cref="BitmapInterpolationMode"/> property.
+        /// </summary>
+        public static readonly StyledProperty<BitmapInterpolationMode> BitmapInterpolationModeProperty =
```

Comment:
There is no such property for TileBrush in WPF. Why do we have it here?

Reply:
Removed